options/locale/locale_zh-TW.ini

```diff
@@ -526,6 +526,7 @@ editor.edit_file=編輯文件
 editor.preview_changes=預覽更改
 editor.cannot_edit_non_text_files=網站介面不能編輯二進位檔案
 editor.edit_this_file=編輯文件
+editor.this_file_locked = 文件已被鎖定
```

Comment:
You should go to crowdin to change this.

Reply:
```suggestion
```

Reply:
This line is removed.